gulp/browserSync.js

```diff
@@ -21,13 +23,47 @@ export function serve(done) {
     // get a fresh copy of the config
     const config = requireUncached(paths.config.themeConfig);
 
-	if (config.dev.browserSync.live) {
-		server.init({
-			proxy: config.dev.browserSync.proxyURL,
-			port: config.dev.browserSync.bypassPort,
-			liveReload: true
-		});
+    // bail early if not serving via BrowserSync
+    if (! config.dev.browserSync.live) {
```

Comment:
> And it should all be moved inside if (config.dev.browserSync.live).

@bamadesigner I took the opposite approach and just exit early if `config.dev.browserSync.live` is not `true`

Reply:
Good call. Let's wrap L39-63 in the check for `if (config.dev.browserSync.https)`, since there's no point if no https.